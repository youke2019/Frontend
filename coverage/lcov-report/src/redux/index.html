<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/redux</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="..\..\prettify.css" />
    <link rel="stylesheet" href="..\..\base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(..\..\sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="..\..\index.html">All files</a> src/redux
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">44.05% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>37/84</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">36% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>9/25</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">25% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>5/20</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">50% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>36/72</span>
      </div>
    </div>
    <p class="quiet">
      Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
    </p>
  </div>
  <div class='status-line low'></div>
<div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file medium" data-value="actions.js"><a href="actions.js.html">actions.js</a></td>
	<td data-value="69.7" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 69%;"></div><div class="cover-empty" style="width:31%;"></div></div></td>
	<td data-value="69.7" class="pct medium">69.7%</td>
	<td data-value="33" class="abs medium">23/33</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="9.09" class="pct low">9.09%</td>
	<td data-value="11" class="abs low">1/11</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="22" class="abs high">22/22</td>
	</tr>

<tr>
	<td class="file low" data-value="reducers.js"><a href="reducers.js.html">reducers.js</a></td>
	<td data-value="24.49" class="pic low"><div class="chart"><div class="cover-fill" style="width: 24%;"></div><div class="cover-empty" style="width:76%;"></div></div></td>
	<td data-value="24.49" class="pct low">24.49%</td>
	<td data-value="49" class="abs low">12/49</td>
	<td data-value="36" class="pct low">36%</td>
	<td data-value="25" class="abs low">9/25</td>
	<td data-value="44.44" class="pct low">44.44%</td>
	<td data-value="9" class="abs low">4/9</td>
	<td data-value="25" class="pct low">25%</td>
	<td data-value="48" class="abs low">12/48</td>
	</tr>

<tr>
	<td class="file high" data-value="state.js"><a href="state.js.html">state.js</a></td>
	<td data-value="100" class="pic high"><div class="chart"><div class="cover-fill cover-full" style="width: 100%;"></div><div class="cover-empty" style="width:0%;"></div></div></td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="2" class="abs high">2/2</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="2" class="abs high">2/2</td>
	</tr>

</tbody>
</table>
</div><div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Thu Aug 01 2019 01:16:28 GMT+0800 (GMT+08:00)
</div>
</div>
<script src="..\..\prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="..\..\sorter.js"></script>
<script src="..\..\block-navigation.js"></script>
</body>
</html>
