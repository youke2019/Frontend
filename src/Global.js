import Storage from 'react-native-storage'
import AsyncStorage from '@react-native-community/async-storage'

let storage = new Storage({
    size: 1000,
    storageBackend: AsyncStorage,
    defaultExpires: 1000 * 60 * 60 * 24, //1 day
    enableCache: true,
    sync: {}
})

global.storage = storage
global.baseUrl = 'http://192.168.42.146:8080'