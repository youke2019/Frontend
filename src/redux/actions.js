export const UPDATE_USER_INFO = 'UPDATEUSERINFO'
export const LOAD_USER_INFO = 'LOADUSERINFO'

export const updateUserInfo = (data) => ({
    type:UPDATE_USER_INFO,
    data
})

export const loadUserInfo = (data) => ({
    type:LOAD_USER_INFO,
    data
})

