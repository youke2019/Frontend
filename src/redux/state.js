const initialState = {
    user_info: null,
    course_list: null,
    login_ready:false,
    sortlist:[],
}

export const mockState = {
    course_list: [
        {"course_id":"GA312","course_name":"花卉学","course_hours":48,"course_credits":3,"general":false,"general_type":"","course_deptname":"设计学院"},
        {"course_id":"GA407","course_name":"花卉应用学","course_hours":32,"course_credits":2,"general":false,"general_type":"","course_deptname":"设计学院"},
        {"course_id":"PA302","course_name":"花卉与草虫禽鸟","course_hours":48,"course_credits":3,"general":false,"general_type":"","course_deptname":"媒体与传播学院"},
        {"course_id":"PL004","course_name":"插花艺术","course_hours":16,"course_credits":1,"general":false,"general_type":"","course_deptname":"设计学院"},
        {"course_id":"PL021","course_name":"花卉艺术","course_hours":32,"course_credits":2,"general":false,"general_type":"","course_deptname":"农业与生物学院"},
        {"course_id":"PL902","course_name":"国花、市花与文化","course_hours":32,"course_credits":2,"general":true,"general_type":"人文学科","course_deptname":"设计学院"},
        {"course_id":"SP079","course_name":"国花、市花鉴赏","course_hours":32,"course_credits":2,"general":false,"general_type":"","course_deptname":"设计学院"}
        ]
}

export default initialState