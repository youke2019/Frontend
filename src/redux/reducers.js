import { combineReducers } from 'redux'
import {
    UPDATE_USER_INFO,
    LOAD_USER_INFO,
    CLEAR_USER_INFO,
    SEARCH_COURSES,
} from "./actions.js"
import initialState from "./state";
import axios from 'axios'


function userInfoReducer(state=initialState.user_info, action) {
    switch (action.type) {
        case UPDATE_USER_INFO:
            storage.save({
                key: 'user',
                data: action.data
            })
            state = action.data
            return state
        case LOAD_USER_INFO:
            state = action.data
            return state
        case CLEAR_USER_INFO:
            state = null
            return state
        default:
            return state
    }
}

function courseReducer(state=initialState.course_list, action) {
    switch (action.type) {
        case SEARCH_COURSES:
            state = action.data
            return state
        default:
            return state
    }
}

export const combinedReducer = combineReducers({
    user_info : userInfoReducer,
    course_list: courseReducer,
})