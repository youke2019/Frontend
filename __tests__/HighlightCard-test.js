import renderer from 'react-test-renderer';
import Global from '../src/Global'
import { createStore } from 'redux'
import { combinedReducer } from '../src/redux/reducers'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16/build'
import Highlight from '../src/pages/Highlight'
import React from 'react'
import HighlightCard from '../src/components/HighlightCard'

const initialState = {
  user_info: {
    id: "30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A",
  },
  course_list: null,
  login_ready:false,
}
const store = createStore(combinedReducer,initialState)
configure({adapter: new Adapter()})



jest.mock("../src/utils/DataRequest.js")

const refresh = jest.fn(()=>{})
const data = JSON.parse("{\"video_id\":1,\"user_id\":\"46420\",\"post_time\":\"177IG770\",\"post_text\":\"UC4GR662312XE4H5MBDU923644WLL3QE563EEQ7QBBT7\",\"video_url\":\"58850\",\"video_type\":\"W\",\"image_url\":\"3\",\"isbanned\":false,\"video_praise_point\":1,\"current_user_praise\":true,\"courseMomentCommentList\":[{\"video_comment_id\":2,\"video_id\":1,\"user_id\":\"14963\",\"video_comment_content\":\"07335\",\"video_comment_time\":\"69660\",\"isbanned\":true},{\"video_comment_id\":3,\"video_id\":1,\"user_id\":\"14963\",\"video_comment_content\":\"16292\",\"video_comment_time\":\"71693\",\"isbanned\":true},{\"video_comment_id\":6,\"video_id\":1,\"user_id\":\"34992\",\"video_comment_content\":\"74875\",\"video_comment_time\":\"95459\",\"isbanned\":true},{\"video_comment_id\":13,\"video_id\":1,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_comment_content\":\"还是是你的\",\"video_comment_time\":\"2019-07-20 23:22:26\",\"isbanned\":false},{\"video_comment_id\":14,\"video_id\":1,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_comment_content\":\"好的好的姐姐的\",\"video_comment_time\":\"2019-07-20 23:38:58\",\"isbanned\":false},{\"video_comment_id\":15,\"video_id\":1,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_comment_content\":\"防脱发头发\",\"video_comment_time\":\"2019-07-20 23:40:46\",\"isbanned\":false},{\"video_comment_id\":16,\"video_id\":1,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_comment_content\":\"刚才填充图册填充图册太吓人饭堂吃同学人吃肉馅肉馅长途车如此\",\"video_comment_time\":\"2019-07-20 23:42:05\",\"isbanned\":false},{\"video_comment_id\":18,\"video_id\":1,\"user_id\":\"1A7A1F80-59D0-4E8C-B4C6-AAAFFC606120\",\"video_comment_content\":\"你是真的牛逼\",\"video_comment_time\":\"2019-07-22 11:32:41\",\"isbanned\":false},{\"video_comment_id\":20,\"video_id\":1,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_comment_content\":\"？\",\"video_comment_time\":\"2019-07-22 16:54:50\",\"isbanned\":false}],\"courseMomentPraiseList\":[{\"video_praise_id\":11,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":12,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":13,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":14,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":15,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":16,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":17,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":18,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":19,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":20,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":21,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":22,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":23,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":24,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":25,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1},{\"video_praise_id\":26,\"user_id\":\"30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A\",\"video_id\":1}]}")
const user_id = "30673CFC-4D4C-4FC1-9FE4-000B3D9DAA9A";

test('renders highlightCard  correctly', (done) => {
  const tree = renderer.create(
    <HighlightCard store = {store} refresh={refresh} data = {data} user_id = {user_id}/>
  ).toJSON();
  setTimeout(()=>{
    expect(tree).toMatchSnapshot();
    done();
  },1000)
});

describe("<HighlightCard /> component",()=>{
  const wrapper = shallow(
    <HighlightCard store = {store} refresh={refresh} data = {data} user_id = {user_id}/>
  );
  const instance = wrapper.dive().dive().instance();
  console.log(instance)
  it("click praise 4 times",(done)=>{
    instance.onPressLike();
    instance.onPressLike();
    instance.onPressLike();
    instance.onPressLike();
    setTimeout(()=>{done()},1000)
  })
  it("send comment request 2 times ",(done)=>{
    instance.sendComment("123123123");
    instance.sendComment("123123123");
    setTimeout(()=>{done()},1000)
  })
  it("send comment when reply done", (done)=>{
    instance.onReplyDone("123123123");
    setTimeout(()=>{done()},1000)
  })
  it("set state when press comment", (done)=>{
    instance.onPressComment();
    setTimeout(()=>{done()},1000)
  })
  it("set state when close comment",(done)=>{
    instance.closeComment()
    setTimeout(()=>{done()},1000)
  })
  it("react when receive props",(done)=>{
    instance.componentWillReceiveProps({
      data:{
        current_user_praise:true,
      }
    },{})
    setTimeout(()=>{done()},1000)
  })
  it("can fold and unfold",(done)=>{
    instance._renderRevealedFooter();
    instance._renderTruncatedFooter();
    instance._handleTextReady();
    setTimeout(()=>{done()},1000)
  })
})