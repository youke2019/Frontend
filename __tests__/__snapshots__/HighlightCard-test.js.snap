// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders highlightCard  correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-start",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
      "marginHorizontal": 20,
      "marginVertical": 5,
      "paddingTop": 10,
    }
  }
>
  <Modal
    animationType="slide"
    hardwareAccelerated={false}
    style={
      Object {
        "flexDirection": "column",
        "justifyContent": "center",
      }
    }
    transparent={true}
    visible={false}
  >
    <View
      accessible={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
    />
    <View
      style={
        Object {
          "borderColor": "lightgrey",
          "borderTopWidth": 0.2,
          "bottom": 0,
          "height": 50,
          "paddingHorizontal": 10,
          "paddingVertical": 0,
          "position": "absolute",
          "width": "100%",
        }
      }
    >
      <RCTScrollView
        keyboardShouldPersistTaps="always"
        style={
          Object {
            "contentSize": 0,
          }
        }
      >
        <View>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "whitesmoke",
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                Object {
                  "borderColor": "orange",
                  "borderRadius": 8,
                  "borderWidth": 0.5,
                  "flex": 8,
                  "flexDirection": "row",
                  "justifyContent": "flex-start",
                  "marginVertical": 10,
                  "paddingHorizontal": 10,
                }
              }
            >
              <TextInput
                allowFontScaling={true}
                autoFocus={true}
                onChangeText={[Function]}
                onSubmitEditing={[Function]}
                placeholder="点击输入回复"
                rejectResponderTermination={true}
                style={
                  Object {
                    "fontSize": 15,
                    "padding": 2,
                    "textAlign": "left",
                    "width": "100%",
                  }
                }
                underlineColorAndroid="transparent"
                value=""
              />
            </View>
            <View
              accessible={true}
              isTVSelectable={true}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "flex": 1.5,
                  "marginVertical": 0,
                  "opacity": 1,
                }
              }
            >
              <View
                style={
                  Object {
                    "backgroundColor": "#FDD326",
                    "borderRadius": 8,
                    "height": "auto",
                    "marginHorizontal": 8,
                    "paddingVertical": 8,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "white",
                      "fontFamily": "字魂70号-灵悦黑体",
                      "fontSize": 15,
                      "lineHeight": 15,
                    }
                  }
                  theme={
                    Object {
                      "colors": Object {
                        "disabled": "hsl(208, 8%, 90%)",
                        "divider": "#bcbbc1",
                        "error": "#ff190c",
                        "grey0": "#393e42",
                        "grey1": "#43484d",
                        "grey2": "#5e6977",
                        "grey3": "#86939e",
                        "grey4": "#bdc6cf",
                        "grey5": "#e1e8ee",
                        "greyOutline": "#bbb",
                        "platform": Object {
                          "android": Object {
                            "error": "#f44336",
                            "primary": "#2196f3",
                            "secondary": "#9C27B0",
                            "success": "#4caf50",
                            "warning": "#ffeb3b",
                          },
                          "ios": Object {
                            "error": "#ff3b30",
                            "primary": "#007aff",
                            "secondary": "#5856d6",
                            "success": "#4cd964",
                            "warning": "#ffcc00",
                          },
                        },
                        "primary": "#2089dc",
                        "searchBg": "#303337",
                        "secondary": "#8F0CE8",
                        "success": "#52c41a",
                        "warning": "#faad14",
                      },
                    }
                  }
                >
                   发送
                </Text>
              </View>
            </View>
          </View>
        </View>
      </RCTScrollView>
    </View>
  </Modal>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexDirection": "column",
        "justifyContent": "flex-start",
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "default_avatar_1",
        }
      }
      style={
        Object {
          "borderRadius": 20,
          "height": 40,
          "width": 40,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "column",
        "height": "100%",
        "marginHorizontal": 10,
        "paddingTop": 15,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "justifyContent": "space-between",
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 15,
            "fontWeight": "bold",
            "width": 120,
          }
        }
        theme={
          Object {
            "colors": Object {
              "disabled": "hsl(208, 8%, 90%)",
              "divider": "#bcbbc1",
              "error": "#ff190c",
              "grey0": "#393e42",
              "grey1": "#43484d",
              "grey2": "#5e6977",
              "grey3": "#86939e",
              "grey4": "#bdc6cf",
              "grey5": "#e1e8ee",
              "greyOutline": "#bbb",
              "platform": Object {
                "android": Object {
                  "error": "#f44336",
                  "primary": "#2196f3",
                  "secondary": "#9C27B0",
                  "success": "#4caf50",
                  "warning": "#ffeb3b",
                },
                "ios": Object {
                  "error": "#ff3b30",
                  "primary": "#007aff",
                  "secondary": "#5856d6",
                  "success": "#4cd964",
                  "warning": "#ffcc00",
                },
              },
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#8F0CE8",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          }
        }
      >
         
        
      </Text>
      <Text>
         
        177IG770
         
      </Text>
    </View>
    <View
      style={
        Object {
          "paddingVertical": 5,
        }
      }
    >
      <View>
        <Text
          numberOfLines={0}
        >
          <Text
            style={
              Object {
                "color": "black",
                "fontSize": 18,
                "fontWeight": "100",
                "letterSpacing": 1,
                "lineHeight": 20,
              }
            }
          >
            UC4GR662312XE4H5MBDU923644WLL3QE563EEQ7QBBT7
          </Text>
        </Text>
      </View>
      <View />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "space-between",
          "paddingHorizontal": 10,
          "paddingVertical": 10,
          "width": "100%",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "opacity": 1,
          }
        }
      >
        <Image
          source={
            Object {
              "uri": "comment_yellow",
            }
          }
          style={
            Object {
              "height": 30,
              "marginRight": 5,
              "width": 30,
            }
          }
        />
        <Text>
          评论
        </Text>
      </View>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "opacity": 1,
          }
        }
      >
        <Image
          source={
            Object {
              "uri": "like_yellow",
            }
          }
          style={
            Object {
              "height": 30,
              "marginRight": 5,
              "width": 30,
            }
          }
        />
        <Text
          style={
            Object {
              "color": "orange",
              "fontSize": 15,
            }
          }
        >
          16
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "whitesmoke",
          "borderRadius": 5,
        }
      }
    >
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <Text
          style={
            Object {
              "fontSize": 14,
              "fontWeight": "200",
            }
          }
          theme={
            Object {
              "colors": Object {
                "disabled": "hsl(208, 8%, 90%)",
                "divider": "#bcbbc1",
                "error": "#ff190c",
                "grey0": "#393e42",
                "grey1": "#43484d",
                "grey2": "#5e6977",
                "grey3": "#86939e",
                "grey4": "#bdc6cf",
                "grey5": "#e1e8ee",
                "greyOutline": "#bbb",
                "platform": Object {
                  "android": Object {
                    "error": "#f44336",
                    "primary": "#2196f3",
                    "secondary": "#9C27B0",
                    "success": "#4caf50",
                    "warning": "#ffeb3b",
                  },
                  "ios": Object {
                    "error": "#ff3b30",
                    "primary": "#007aff",
                    "secondary": "#5856d6",
                    "success": "#4cd964",
                    "warning": "#ffcc00",
                  },
                },
                "primary": "#2089dc",
                "searchBg": "#303337",
                "secondary": "#8F0CE8",
                "success": "#52c41a",
                "warning": "#faad14",
              },
            }
          }
        >
           
          
        </Text>
        <Text
          style={
            Object {
              "fontSize": 14,
              "lineHeight": 18,
              "width": "79%",
            }
          }
        >
          : 还是是你的
        </Text>
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <Text
          style={
            Object {
              "fontSize": 14,
              "fontWeight": "200",
            }
          }
          theme={
            Object {
              "colors": Object {
                "disabled": "hsl(208, 8%, 90%)",
                "divider": "#bcbbc1",
                "error": "#ff190c",
                "grey0": "#393e42",
                "grey1": "#43484d",
                "grey2": "#5e6977",
                "grey3": "#86939e",
                "grey4": "#bdc6cf",
                "grey5": "#e1e8ee",
                "greyOutline": "#bbb",
                "platform": Object {
                  "android": Object {
                    "error": "#f44336",
                    "primary": "#2196f3",
                    "secondary": "#9C27B0",
                    "success": "#4caf50",
                    "warning": "#ffeb3b",
                  },
                  "ios": Object {
                    "error": "#ff3b30",
                    "primary": "#007aff",
                    "secondary": "#5856d6",
                    "success": "#4cd964",
                    "warning": "#ffcc00",
                  },
                },
                "primary": "#2089dc",
                "searchBg": "#303337",
                "secondary": "#8F0CE8",
                "success": "#52c41a",
                "warning": "#faad14",
              },
            }
          }
        >
           
          
        </Text>
        <Text
          style={
            Object {
              "fontSize": 14,
              "lineHeight": 18,
              "width": "79%",
            }
          }
        >
          : 好的好的姐姐的
        </Text>
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <Text
          style={
            Object {
              "fontSize": 14,
              "fontWeight": "200",
            }
          }
          theme={
            Object {
              "colors": Object {
                "disabled": "hsl(208, 8%, 90%)",
                "divider": "#bcbbc1",
                "error": "#ff190c",
                "grey0": "#393e42",
                "grey1": "#43484d",
                "grey2": "#5e6977",
                "grey3": "#86939e",
                "grey4": "#bdc6cf",
                "grey5": "#e1e8ee",
                "greyOutline": "#bbb",
                "platform": Object {
                  "android": Object {
                    "error": "#f44336",
                    "primary": "#2196f3",
                    "secondary": "#9C27B0",
                    "success": "#4caf50",
                    "warning": "#ffeb3b",
                  },
                  "ios": Object {
                    "error": "#ff3b30",
                    "primary": "#007aff",
                    "secondary": "#5856d6",
                    "success": "#4cd964",
                    "warning": "#ffcc00",
                  },
                },
                "primary": "#2089dc",
                "searchBg": "#303337",
                "secondary": "#8F0CE8",
                "success": "#52c41a",
                "warning": "#faad14",
              },
            }
          }
        >
           
          
        </Text>
        <Text
          style={
            Object {
              "fontSize": 14,
              "lineHeight": 18,
              "width": "79%",
            }
          }
        >
          : 防脱发头发
        </Text>
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <Text
          style={
            Object {
              "fontSize": 14,
              "fontWeight": "200",
            }
          }
          theme={
            Object {
              "colors": Object {
                "disabled": "hsl(208, 8%, 90%)",
                "divider": "#bcbbc1",
                "error": "#ff190c",
                "grey0": "#393e42",
                "grey1": "#43484d",
                "grey2": "#5e6977",
                "grey3": "#86939e",
                "grey4": "#bdc6cf",
                "grey5": "#e1e8ee",
                "greyOutline": "#bbb",
                "platform": Object {
                  "android": Object {
                    "error": "#f44336",
                    "primary": "#2196f3",
                    "secondary": "#9C27B0",
                    "success": "#4caf50",
                    "warning": "#ffeb3b",
                  },
                  "ios": Object {
                    "error": "#ff3b30",
                    "primary": "#007aff",
                    "secondary": "#5856d6",
                    "success": "#4cd964",
                    "warning": "#ffcc00",
                  },
                },
                "primary": "#2089dc",
                "searchBg": "#303337",
                "secondary": "#8F0CE8",
                "success": "#52c41a",
                "warning": "#faad14",
              },
            }
          }
        >
           
          
        </Text>
        <Text
          style={
            Object {
              "fontSize": 14,
              "lineHeight": 18,
              "width": "79%",
            }
          }
        >
          : 刚才填充图册填充图册太吓人饭堂吃同学人吃肉馅肉馅长途车如此
        </Text>
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <Text
          style={
            Object {
              "fontSize": 14,
              "fontWeight": "200",
            }
          }
          theme={
            Object {
              "colors": Object {
                "disabled": "hsl(208, 8%, 90%)",
                "divider": "#bcbbc1",
                "error": "#ff190c",
                "grey0": "#393e42",
                "grey1": "#43484d",
                "grey2": "#5e6977",
                "grey3": "#86939e",
                "grey4": "#bdc6cf",
                "grey5": "#e1e8ee",
                "greyOutline": "#bbb",
                "platform": Object {
                  "android": Object {
                    "error": "#f44336",
                    "primary": "#2196f3",
                    "secondary": "#9C27B0",
                    "success": "#4caf50",
                    "warning": "#ffeb3b",
                  },
                  "ios": Object {
                    "error": "#ff3b30",
                    "primary": "#007aff",
                    "secondary": "#5856d6",
                    "success": "#4cd964",
                    "warning": "#ffcc00",
                  },
                },
                "primary": "#2089dc",
                "searchBg": "#303337",
                "secondary": "#8F0CE8",
                "success": "#52c41a",
                "warning": "#faad14",
              },
            }
          }
        >
           
          
        </Text>
        <Text
          style={
            Object {
              "fontSize": 14,
              "lineHeight": 18,
              "width": "79%",
            }
          }
        >
          : 你是真的牛逼
        </Text>
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <Text
          style={
            Object {
              "fontSize": 14,
              "fontWeight": "200",
            }
          }
          theme={
            Object {
              "colors": Object {
                "disabled": "hsl(208, 8%, 90%)",
                "divider": "#bcbbc1",
                "error": "#ff190c",
                "grey0": "#393e42",
                "grey1": "#43484d",
                "grey2": "#5e6977",
                "grey3": "#86939e",
                "grey4": "#bdc6cf",
                "grey5": "#e1e8ee",
                "greyOutline": "#bbb",
                "platform": Object {
                  "android": Object {
                    "error": "#f44336",
                    "primary": "#2196f3",
                    "secondary": "#9C27B0",
                    "success": "#4caf50",
                    "warning": "#ffeb3b",
                  },
                  "ios": Object {
                    "error": "#ff3b30",
                    "primary": "#007aff",
                    "secondary": "#5856d6",
                    "success": "#4cd964",
                    "warning": "#ffcc00",
                  },
                },
                "primary": "#2089dc",
                "searchBg": "#303337",
                "secondary": "#8F0CE8",
                "success": "#52c41a",
                "warning": "#faad14",
              },
            }
          }
        >
           
          
        </Text>
        <Text
          style={
            Object {
              "fontSize": 14,
              "lineHeight": 18,
              "width": "79%",
            }
          }
        >
          : ？
        </Text>
      </View>
    </View>
  </View>
</View>
`;
